lib/phraseapp_updater/phraseapp_api.rb

```diff
@@ -174,7 +174,12 @@ def verify_uploads(uploads)
           end
         end
 
-        sleep(2) unless uploads.empty?
```

Comment:
https://developers.phrase.com/api/#overview--rate-limiting

In theory you can make 1 request every 3.333 seconds and not hit the rate limit. So I think if you actually just bumped this line to 4 it would maintain okay. 

This says "per user" so I'm not sure if they really mean per client.

Reply:
> you actually just bumped this line to 4 it would maintain okay

This is doing up to a request per language each time around the loop, so it's a bit multiplied out.

> This says "per user" so I'm not sure if they really mean per client.

It's a complete assumption, but I was assuming they meant "per API key".